apps/condo/domains/organization/integrations/sbbol/sync/syncUser.js

```diff
@@ -68,42 +80,60 @@ const cleanEmailForAlreadyExistingUserWithGivenEmail = async ({ email, userIdToE
  * @return {Promise<{importId}|*>}
  */
 const syncUser = async ({ context, userInfo }) => {
-    const returnFields = 'id phone email name importId importRemoteSystem'
-    const importFields = {
-        type: 'staff',
-        importId: userInfo.importId,
-        importRemoteSystem: userInfo.importRemoteSystem,
+    const returnFields = 'id phone email name'
+    const identityWhereStatement = {
+        user: { type: 'staff' },
+        identityId: userInfo.importId,
+        identityType: SBBOL_IDP_TYPE,
     }
-    const userFields = {
+    const userWhereStatement = {
         type: 'staff',
         phone: userInfo.phone,
     }
-    const existingUsers = await getItems({
+
+    // let's search users by UserExternalIdentity and phone
+    const importedUsers = (await getItems({
```

Comment:
it's better not to use `@keystonejs/server-side-graphql-client`, if you can, try to use our server util

Reply:
+

Reply:
Fixed